packages/server/lib/project.js

```diff
@@ -581,11 +581,15 @@ class Project extends EE {
     // and example support file if dir doesnt exist
     push(scaffold.support(cfg.supportFolder, cfg))
 
-    // if we're in headed mode add these other scaffolding
-    // tasks
-    if (!cfg.isTextTerminal) {
+    // if we're in headed mode add these other scaffolding tasks
+    const scaffoldExamples = !cfg.isTextTerminal || process.env.CYPRESS_INTERNAL_FORCE_SCAFFOLD
+
+    if (scaffoldExamples) {
+      debug('will scaffold integration and fixtures folder')
```

Comment:
Just being more specific since it scaffolds plugin and support folder/files too. A bit nitpicky.

```suggestion
      debug('will scaffold cypress folders and files')
```

Reply:
so notice that it scaffolds support file _already_. I have not changed it, there is a comment above that this is historical artifact. I only added scaffolding of the integration and fixture folders, so that's why the debug message was explicit

Reply:
well that's weird